### ISSUE TITLE
Missing image legend with animated gif

### ISSUE THREAD

**Issue by ImaCrea** [h-enk/doks#634]
## Description

title is pretty clear

### Steps to reproduce

Insert an animated gif with image code like this:
```![Démonstration de l'annuaire](annuaire.gif "Vous pouvez faire aussi une recherche depuis l'annuaire pour filtrer les résultats et trouver plus rapidement la personne.")  ```

### Expected result

Legend should appear underneath the image

### Actual result

No legend displayed

### Environment

```
> doks-child-theme@0.3.5 precheck
> npm version

{
  'doks-child-theme': '0.3.5',
  npm: '8.1.4',
  node: '17.2.0',
  v8: '9.6.180.14-node.12',
  uv: '1.42.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '102',
  nghttp2: '1.45.1',
  napi: '8',
  llhttp: '6.0.4',
  openssl: '3.0.0+quic',
  cldr: '40.0',
  icu: '70.1',
  tz: '2021a3',
  unicode: '14.0',
  ngtcp2: '0.1.0-DEV',
  nghttp3: '0.1.0-DEV'
}

> doks-child-theme@0.3.5 check
> exec-bin node_modules/.bin/hugo/hugo version

hugo v0.88.1-5BC54738+extended linux/amd64 BuildDate=2021-09-04T09:39:19Z VendorInfo=gohugoio
npm notice 
npm notice New minor version of npm available! 8.1.4 -> 8.3.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.3.2
npm notice Run npm install -g npm@8.3.2 to update!
npm notice 
```

**Comment by ImaCrea** [comment]
thanks!!! 🤗

**Comment by h-enk** [comment]
It looks like the width of your gif is smaller than the `smallLimit` set in `./config/_default/params.toml`.

An [`img`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) element is generated for small images, a [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) element — with optional a `<figcaption>` element — for large images. 

You can set `smallLimit` in `./config/_default/params.toml`, e.g. `smallLimit = "420"`.

**Comment by ImaCrea** [comment]
Thanks, I see if I follow your point yes it's working again.

But because of your answer [here](https://github.com/h-enk/doks/discussions/597) about gif not playing I put "1500" for smallLimit actually.

I'm a bit blocked, it looks like I can't have both world I'm afraid.

**Comment by h-enk** [comment]
> I'm a bit blocked, it looks like I can't have both world I'm afraid.

Yeah, correct.  For now, maybe you could mimic the `<figcaption>` element, and add something like this below the small image gif (downside: html in markdown):

```md
<p class="figure-caption">Day and Night, 1938 — M.C. Escher</p>
``` 

For Doks v0.5.0, I've updated https://github.com/h-enk/doks/issues/502 w/ "Add support for animated GIF with caption" — see also: [Animated GIF with caption](https://developers.facebook.com/docs/instant-articles/reference/animated-gif/#examples-caption)
